.github/workflows/notify_team_new_comment.yml

```diff
@@ -0,0 +1,37 @@
+name: Send a slack notification when a contributor comments on issue
+
+on:
+  workflow_call:
+    secrets:
+      SLACK_WEBHOOK_URL:
+        required: true
+
+jobs:
+  contributor_issue_comment:
+    name: Contributor issue comment
+
+    if: >-
+      ${{
+        !github.event.issue.pull_request &&
+        github.event.comment.author_association != 'MEMBER' &&
+        github.event.comment.author_association != 'OWNER'
+      }}
+
+    runs-on: ubuntu-latest
+    steps:
+      - name: Escape title double quotes
+        id: escape_title
+        env:
+          ISSUE_TITLE: ${{ github.event.issue.title }}
+        run: echo "ISSUE_TITLE=${ISSUE_TITLE//\"/\\\"}" >> "$GITHUB_OUTPUT"
+
+      - name: Send message to Slack channel
+        env:
+            SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+            SLACK_WEBHOOK_TYPE: INCOMING_WEBHOOK
+        uses: slackapi/slack-github-action@v1.27.0
```

Comment:
Hi @rparadowski - I was wondering if you'd be willing to take on upgrading the action to v2 in the course of this work?

No obligation here, but I suspect we might have to tweak how we invoke the action based on their most recent changes and thought doing it in the course of this work might be feasible: https://github.com/slackapi/slack-github-action/releases/tag/v2.0.0

Reply:
@rparadowski I invited my colleague @rtibbles here because it seemed you're on search for another task and this shared action is just on time :)! Otherwise we'd need to do this in three repos.

And after I will have done the testing, @rtibbles will join us for more detailed code review - he's one of the most knowledgeable in this regard on the team.

Thanks both.

Reply:
Hi, version upgraded and tested:
![image](https://github.com/user-attachments/assets/023beaa3-8aa5-4e5a-a58d-2a0db1cf6924)

Reply:
@MisRob @rtibbles ^^

Reply:
Thank you @rparadowski, much appreciated!